src/llmcompressor/modifiers/utils/hooks.py

```diff
@@ -29,18 +29,29 @@ class HooksMixin(BaseModel):
         - modifier.remove_hooks()
     """
 
-    _HOOKS_DISABLED: ClassVar[bool] = False  # attached to global HooksMixin
-    _hooks: Set[RemovableHandle] = set()  # attached to local subclasses
+    # attached to global HooksMixin class
```

Comment:
Can you update the docstring to show how to enable or disable a subset of the hooks?

Reply:
Done